{
    "id": 126050,
    "name": "CVE-2019-5629",
    "cwe": "CWE-427",
    "modified": "2020-10-16T13:10:11",
    "published": "2019-07-12T23:15:10",
    "status": "Analyzed",
    "summary": "Rapid7 Insight Agent, version 2.6.3 and prior, suffers from a local privilege escalation due to an uncontrolled DLL search path. Specifically, when Insight Agent 2.6.3 and prior starts, the Python interpreter attempts to load python3.dll at \"C:\\DLLs\\python3.dll,\" which normally is writable by locally authenticated users. Because of this, a malicious local user could use Insight Agent's startup conditions to elevate to SYSTEM privileges. This issue was fixed in Rapid7 Insight Agent 2.6.4.",
    "cvss3": {
        "cvss": 7.8,
        "exploitability_score": 1.8,
        "impact_score": 5.9,
        "vector": "CVSS:3.1/AV:L/AC:L/PR:L/UI:N/S:U/C:H/I:H/A:H"
    },
    "cvss2": {
        "exploitability_score": 3.9,
        "impact_score": 10.0,
        "cvss": 7.2,
        "vector": "AV:L/AC:L/Au:N/C:C/I:C/A:C"
    },
    "cpes": [
        {
            "id": 296583
        },
        {
            "id": 296584
        },
        {
            "id": 296585
        },
        {
            "id": 296586
        },
        {
            "id": 296587
        },
        {
            "id": 296588
        },
        {
            "id": 866635
        }
    ]
}